common/fileview2/fvresultset.cpp

```diff
@@ -146,8 +146,9 @@ IFvDataSource * createFileDataSource(IDistributedFile * df, const char * logical
         const char * kind = properties.queryProp("@kind");
         if (kind && (stricmp(kind, "key") == 0))
         {
-            if (isSimplifiedRecord(diskRecord, true))
-                ds.setown(new IndexDataSource(logicalName, diskRecord, username, password));
+            OwnedHqlExpr indexRecord = annotateIndexBlobs(diskRecord);
```

Comment:
Marking / Protecting the blobs from being force simplified in getFileViewRecord().. other uses of that function may still rely on simplification of blobs.  Otherwise I would just never simplify.